src/Tribe/Views/V2/Views/By_Day_View.php

```diff
@@ -256,16 +257,16 @@ public function get_grid_days( $date = null, $force = false ) {
 
 		$all_day_event_ids = [];
 		$site_timezone = Timezones::build_timezone_object();
-		$use_site_timezone = Timezones::is_mode( 'site' );
 		$utc = Timezones::build_timezone_object( 'UTC' );
 
 		// phpcs:ignore
-		/** @var \DateTime $day */
+		/** @var \Tribe\Utils\Date_I18n $day */
 		foreach ( $days as $day ) {
 			$day_string = $day->format( 'Y-m-d' );
 
 			if ( $using_period_repository && isset( $repository ) ) {
 				$day_results = $repository->by_date( $day_string )->get_set();
+				$day_event_ids = [];
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: dc7f4dab465a36d320ad7380f42feb68</sub>